lib/xapi-stdext-unix/unixext_stubs.c

```diff
@@ -334,17 +334,17 @@ CAMLprim value stub_statvfs(value filename)
 	Field(v,i)=tmp;
   }
 
-  Field(v,0)=caml_copy_int64(buf.f_bsize);
-  Field(v,1)=caml_copy_int64(buf.f_frsize);
-  Field(v,2)=caml_copy_int64(buf.f_blocks);
-  Field(v,3)=caml_copy_int64(buf.f_bfree);
-  Field(v,4)=caml_copy_int64(buf.f_bavail);
-  Field(v,5)=caml_copy_int64(buf.f_files);
-  Field(v,6)=caml_copy_int64(buf.f_ffree);
-  Field(v,7)=caml_copy_int64(buf.f_favail);
-  Field(v,8)=caml_copy_int64(buf.f_fsid);
-  Field(v,9)=caml_copy_int64(buf.f_flag);
-  Field(v,10)=caml_copy_int64(buf.f_namemax);
+  Store_field(v, 0, caml_copy_int64(buf.f_bsize));
```

Comment:
It think it is better to use `caml_alloc` rather than `alloc_small` as well, above. And we need to remove the zeroing right above here.

Reply:
Changed it accordingly and removed the initialization loop.